core/templates/dev/head/pages/about-page/about-page.module.ts

```diff
@@ -32,11 +32,11 @@ import { HttpClientModule } from '@angular/common/http';
 })
 export class ServiceBootstrapComponent {}
 
-import { AppConstants } from 'app.constants';
+import { AppConstants } from '../../app.constants';
```

Comment:
@NoumanMasood, Can you please the reason behind these changes?

Reply:
All constants are not imported correctly. They were giving errors because of wrong path. Now it's fixed

Reply:
I think the [webpack is configured](https://github.com/oppia/oppia/blob/develop/webpack.common.config.ts#L39) (resolve.modules) such that it can look into the files directly. So I don't think this is required!

Reply:
Constants are not configured in webpack. That is why we have to import it from it's correct path otherwise it will give error 'Cannot find module'

Reply:
![type](https://user-images.githubusercontent.com/14115229/67712109-2845c980-f9e5-11e9-9f17-67469057df0c.png)

Reply:
I don't think there is any issue with imports, the issue #7343 states that typescript is throwing an error on accessing dynamically assigned constants of AppConstants. 

Assigning this to @vojtechjelinek for the review!

Reply:
Webpack is configured to resolve the modules, but your editor isn't aware of the configuration @NoumanMasood